website/content/api-docs/system/secrets-sync.mdx

```diff
@@ -424,7 +424,16 @@ This endpoint creates a destination to synchronize action secrets with a GitHub
 
 - `name` `(string: <required>)` - Specifies the name for this destination. This is specified as part of the URL.
 
-- `access_token` `(string: <required>)` - Fine-grained or personal access token.
+- `access_token` `(string: <optional>)` - Fine-grained or personal access token.
+  Use `access_token` as an alternative to authenticating with a GitHub app.
+
+- `app_name` `(string: <optional>)` - The name of a GitHub App to use for
+  authentication. Use `app_name` and `installation_id` as an alternative to
+  authenticating with an access token.
```

Comment:
I think it's worth calling out this is the name that was configured in Vault, the phrasing suggests using the name configured on the GitHub side would work.